### ISSUE TITLE
Add "age" as a column in the earnings list?

### ISSUE THREAD

**Issue by stkeros** [Gregable/social-security-tools#81]
It might be nice to list a person's age to the right of the year on the earnings record input and additional earnings tables.

I think in terms of "what if I work until 55" rather than "what if I work for "55 - current age" more years. But adding age, it would make it easy to see the age at last year of earnings.

**Comment by Gregable** [comment]
Agreed. For consistency, I'll add it to both the earnings record and future earnings.
